docs/content/configuration/index.md

```diff
@@ -820,7 +820,12 @@ An example of compaction config is:
 }
 ```
 
-For realtime dataSources, it's recommended to set `skipOffsetFromLatest` to some sufficiently large value to avoid frequent compact task failures.
+Note that compact tasks can fail if their locks are revoked by other tasks of higher priorities.
+Since realtime tasks have a higher priority than compact task by default,
+it can be problematic if there're frequent conflicts between compact tasks and realtime tasks.
+If this is the case, coordinator's automatic compaction might be stuck because of the frequent compact task fails.
```

Comment:
the coordinator's

Reply:
might get stuck

because of frequent compact task failures

Reply:
Thanks!